docs/operations/customization.md

```diff
@@ -0,0 +1,216 @@
+
+### Example
+
+The following example configuration in `values.yaml` shows some of the possibilities to achieve a custom theming and branding:
+
+```yaml
+global:
+  dashboard:
+    frontendConfig:
+      # ...
+      branding:
+        productLogoUrl: data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNCAyNCI+PHBhdGggZmlsbD0iIzBCODA2MiIgZD0iTTIsMjJWMjBDMiwyMCA3LDE4IDEyLDE4QzE3LDE4IDIyLDIwIDIyLDIwVjIySDJNMTEuMyw5LjFDMTAuMSw1LjIgNCw2LjEgNCw2LjFDNCw2LjEgNC4yLDEzLjkgOS45LDEyLjdDOS41LDkuOCA4LDkgOCw5QzEwLjgsOSAxMSwxMi40IDExLDEyLjRWMTdDMTEuMywxNyAxMS43LDE3IDEyLDE3QzEyLjMsMTcgMTIuNywxNyAxMywxN1YxMi44QzEzLDEyLjggMTMsOC45IDE2LDcuOUMxNiw3LjkgMTQsMTAuOSAxNCwxMi45QzIxLDEzLjYgMjEsNCAyMSw0QzIxLDQgMTIuMSwzIDExLjMsOS4xWiIgLz48L3N2Zz4=
+        productName: Nucleus
+        productTitleSuperscript: false
+        productTitle: Nucleus Live
+        productDescription: Supercool Cluster Service
+        teaserHeight: 160
+        teaserTemplate: |
+          <div
+            class="text-center px-2"
+          >
+            <a
+              href="/"
+              class="text-decoration-none"
+            >
+              <img
+                src="{{ productLogoUrl }}"
+                width="80"
+                height="80"
+                alt="{{ productName }} Logo"
+                class="pointer-events-none"
+              >
+              <div
+                class="font-weight-thin text-grey-lighten-4"
+                style="font-size: 32px; line-height: 32px; letter-spacing: 2px;"
+              >
+                {{ productTitle }}
+              </div>
+              <div class="text-body-1 font-weight-normal text-primary mt-1">
+                {{ productDescription }}
+              </div>
+            </a>
+          </div>
+        loginTeaserHeight: 296
+        loginTeaserTemplate: |
+          <div
+            class="d-flex flex-column align-center justify-center bg-main-background-darken-1 pa-3"
+            style="min-height: {{minHeight}}px"
+          >
+            <img
+              src="{{productLogoUrl}}"
+              alt="Login to {{productName}}"
+              width="140"
+              height="140"
+              class="mt-2"
+            >
+            <div class="text-h3 text-center font-weight-thin text-white mt-4">
+              {{productTitle}}
+            </div>
+            <div class="text-h5 text-center font-weight-light text-primary mt-1">
+              {{productSlogan}}
+            </div>
+          </div>
+        loginFooterTemplate: |
+          <div class="text-anchor text-caption">
+            Copyright 2023 by Nucleus Corporation
+          </div>
+        loginHints:
+          - title: Support
+            href: https://gardener.cloud
+          - title: Documentation
+            href: https://gardener.cloud/docs
+        oidcLoginTitle: MyIDP
+        oidcLoginText: Press LOGIN to be redirected to My Identity Service.
+      themes:
```

Comment:
themes indentation must be same level as branding

Reply:
Hmm. It is the same, isn't it?